Capacitor integration

amrmelsayed opened the pull request.

waleedkadous commented on capacitor.config.json:
```diff
@@ -0,0 +1,5 @@
+{
+  "appId": "com.orgnametbd.ansariapp",
```
chat.ansari.ansariapp?

amrmelsayed replied:
Done, the application identifier is now chat.ansari.app for both Android and iOS

waleedkadous commented on android/app/build.gradle:
```diff
@@ -0,0 +1,54 @@
+apply plugin: 'com.android.application'
```
chat.ansari.application?

amrmelsayed replied:
com.android.application is a native android plugin, the app identifier is set in the applicationId attribute.

waleedkadous commented on android/app/src/androidTest/java/com/getcapacitor/myapp/ExampleInstrumentedTest.java:
Why are we checking in an Example test? Should this be renamed?

amrmelsayed replied:
I will remove the test case, I shouldn't have commited it.

amrmelsayed replied:
File removed

waleedkadous commented on android/app/src/androidTest/java/com/getcapacitor/myapp/ExampleInstrumentedTest.java:
```diff
@@ -0,0 +1,26 @@
+package com.getcapacitor.myapp;
```
Update this?

amrmelsayed replied:
File removed

waleedkadous commented on android/app/src/test/java/com/getcapacitor/myapp/ExampleUnitTest.java:
```diff
@@ -0,0 +1,18 @@
+package com.getcapacitor.myapp;
```
Update this?

amrmelsayed replied:
File removed

waleedkadous reviewed (approved):
Excellent work!

amrmelsayed commented on .env.example:
```diff
@@ -3,13 +3,14 @@ NODE_ENV=production
 REACT_APP_SHARE_URL=http://localhost:3000
+REACT_APP_PROXY_HOST=http://localhost:3000
 
 REACT_APP_API_V2_URL=https://api-beta.ansari.chat/api/v2
```
Done

amrmelsayed commented:
I have updated the application identifier, regenerated the logo and assets using the svg version, removed test case samples and updated .env file.

Let me know if you have any other requests, if you are happy with the changes, please merge them into the main branch as I need to create another major branch to replace React Scripts and Webpack with Expo and Metro.

waleedkadous commented:
Merged. Thank you so much for your help!

On Fri, Jan 31, 2025 at 7:59 PM Amr Elsayed ***@***.***>
wrote:

> I have updated the application identifier, regenerated the logo and assets
> using the svg version, removed test case samples and updated .env file.
>
> Let me know if you have any other requests, if you are happy with the
> changes, please merge them into the main branch as I need to create another
> major branch to replace React Scripts and Webpack with Expo and Metro.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ansari-project/ansari-frontend/pull/19#issuecomment-2628751441>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAUXGUARHPWTMPT6JEBRUBL2NRBCJAVCNFSM6AAAAABV5JRQOOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMMRYG42TCNBUGE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
